src/codegen/mod.rs

```diff
@@ -716,7 +716,9 @@ impl<'a> CodeGenerator for Vtable<'a> {
             .item()
             .pub_()
             .with_attrs(attributes)
```

Comment:
Whoops, you're totally right, you only need to remove the `if` a few lines above this one that pushes the `Default` attr :)